flow/scripts/floorplan.tcl

```diff
@@ -87,6 +87,37 @@ remove_buffers
 
 source $::env(SCRIPTS_DIR)/report_metrics.tcl
 report_metrics "floorplan final"
+set num_instances [llength [get_cells -hier *]]
+puts "number instances before restructure is $num_instances"
+puts "Design Area before restructure"
+report_design_area
```

Comment:
wouldn't it make more sense to put this inside the if statement? It message about restructuring doesn't make sense if that is not enabled.

Reply:
These numbers were required for non-restructure run as well so that when comparison is done, all numbers are there. We can re-word this. Let me think more about it.

Reply:
if the numbers are needed then they are needed. The only change that thought would be needed really, is just to keep the naming consistent for the verilog files that are getting written out.

Reply:
Updated naming convention.